recipe/build-mpi.sh

```diff
@@ -11,6 +11,16 @@ export CC=$(basename "$CC")
 export CXX=$(basename "$CXX")
 export FC=$(basename "$FC")
 
+if [[ $CONDA_BUILD_CROSS_COMPILATION == 1 && $target_platform == osx-arm64 ]]; then
+    # use Conda-Forge's Arm64 config.guess and config.sub, see
+    # https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/
+    list_config_to_patch=$(find ./ -name config.guess | sed -E 's/config.guess//')
+    for config_folder in $list_config_to_patch; do
+        echo "copying config to $config_folder ...\n"
+        cp -v $BUILD_PREFIX/share/gnuconfig/config.* $config_folder
+    done
+fi
+
 ./autogen.sh
```

Comment:
@leofang Do we need to run this unconditionally?

Reply:
Looks like it was added for OS X Arm64. I can move it to the `if` block above it.

Reply:
Done in 06d2e09.